config/api/src/main/java/org/jboss/windup/config/metadata/LabelProviderMetadata.java

```diff
@@ -0,0 +1,6 @@
+package org.jboss.windup.config.metadata;
+
+public interface LabelProviderMetadata extends LabelsetMetadata
```

Comment:
Please add comments to describe the class.

Reply:
done